tests/test_asset_hierarchy.py

```diff
@@ -65,6 +65,8 @@ def test_shotgun_links_sync(self, mocked_sg):
             "code": "Foo",
             "description": "I'm Foo !",
             "tasks": [],
+            SHOTGUN_JIRA_ID_FIELD: issue.key,
+            SHOTGUN_SYNC_IN_JIRA_FIELD: True,
```

Comment:
The following tests are failing because of the new changes on asset_issue_handler.py. 

We need to review.